lms/djangoapps/courseware/courses.py

```diff
@@ -593,3 +593,19 @@ def _get_default_child_module(child_modules):
                 child = _get_default_child_module(children)
 
     return child
+
+
+def get_course_chapters(course_key):
+    """
+    Extracts the chapter block keys from a course structure.
+
+    Arguments:
+        course_key (CourseLocator): The course key
+    Returns:
+        list (string): The list of string representations of the chapter block keys in the course.
+    """
+    try:
+        chapters = modulestore().get_items(course_key, qualifiers={'category': 'chapter'})
```

Comment:
What you're doing here is more technically correct than the previous code, but I think it's safer to grab the root course element and iterate its children like the previous CCX course does because:

* The code above can introduce orphaned chapters into the results (where the chapters exist in the data but aren't actually active children of the course any longer).
* It's likely to be less performant, particularly for large courses, because qualifier filters do a dumb scan of every block in the course for comparisons, and this method will require instantiating more SectionDescriptor blocks (which subclass Sequential, which has a lot of cruft in it).

The modulestore() equivalent of the old CCX code (grabbing `course_struct['blocks'][course_struct['root']].get('children', [])` would be to do:

```python
chapter_keys = modulstore().get_course(course_key).children
```

That would give you a list of `BlockUsageKeys` for the chapters (you could further filter this on type  by looking at the `.block_type` of each key if desired), and then convert to string to get what you're returning below.